package.json

```diff
@@ -10,12 +10,19 @@
     "lint": "eslint . --ext .js,.jsx,.ts,.tsx"
   },
   "dependencies": {
+    "@react-navigation/drawer": "^6.5.0",
+    "@react-navigation/native": "^6.0.13",
     "@rneui/base": "^4.0.0-rc.6",
     "@rneui/themed": "^4.0.0-rc.6",
     "react": "18.1.0",
     "react-devtools": "^4.26.0",
     "react-native": "0.70.1",
-    "react-native-nfc-manager": "^3.14.0"
+    "react-native-gesture-handler": "^2.7.1",
```

Comment:
There are a lot of new imports 
I guess we will use it later ?

Reply:
These are the dependencies for the drawer component